qiskit/circuit/library/standard_gates/u1.py

```diff
@@ -11,6 +11,7 @@
 # that they have been altered from the originals.
 
 """U1 Gate."""
+from __future__ import annotations
 from cmath import exp
 from typing import Optional, Union
```

Comment:
There seems to be some trailing `Optional` and `Union` uses here, do you want to update them in this PR, too? 🙂